src/components/application_manager/test/commands/mobile/create_interaction_choice_set_test.cc

```diff
@@ -213,11 +212,10 @@ TEST_F(CreateInteractionChoiceSetRequestTest, OnEvent_VR_UNSUPPORTED_RESOURCE) {
   utils::SharedPtr<CreateInteractionChoiceSetRequest> req_vr =
       CreateCommand<CreateInteractionChoiceSetRequest>(msg_vr);
 
-  MockAppPtr mock_app = CreateMockApp();
-  ON_CALL(app_mngr_, application(_)).WillByDefault(Return(mock_app));
-  ON_CALL(*mock_app, app_id()).WillByDefault(Return(kConnectionKey));
+  ON_CALL(app_mngr_, application(_)).WillByDefault(Return(app_));
```

Comment:
@AByzhynar 
Quick search revealed more than 1200 place to rename. Auto replacement didn't work and required a lot of manual work and analysis what to rename. Also such changes are huge and out of task scope. Beside that will completely hide the root changes done to fix original issue.